docs/content/docs/dev/datastream/operators/overview.md

```diff
@@ -700,12 +700,12 @@ some_stream.filter(...).name("filter").set_description("x in (1, 2, 3, 4) and y
 {{< /tab >}}
 {{< /tabs>}}
 
-The format of description of a job vertex is a tree format string by default. 
-Users can set `pipeline.vertex-description-mode` to `CASCADING`, if they want to set description to be the cascading format as in former versions.
+Depending on your needs, you may want to modify some of the default naming behaviors for vertices and operators.
 
-Operators generated by Flink SQL will have a name consisted by type of operator and id, and a detailed description, by default.
-Users can set `table.exec.simplify-operator-name-enabled` to `false`, if they want to set name to be the detailed description as in former versions.
+By default, the description of a job vertex is organized in a multi-tree line mode.
+To use a single line cascading mode, set `pipeline.vertex-description-mode` to `CASCADING`
 
-When the topology of the pipeline is complex, users can add a topological index in the name of vertex by set `pipeline.vertex-name-include-index-prefix` to `true`,
-so that we can easily find the vertex in the graph according to logs or metrics tags.
+By default, operators generated by Flink SQL will have a name composed of the operator type and id. Details will remain in the description.
```

Comment:
you mention "Details will remain in the description.". What details are these - it would be better to list them. 
You say remain - I am not sure what we are trying to say with this. I suggest something like:
"The description contains more information on the operator including ...."